environments/aks/sbox.tfvars

```diff
@@ -22,6 +22,11 @@ linux_node_pool = {
   min_nodes = 4,
   max_nodes = 10
 }
+cronjob = {
```

Comment:
I think you would need to specify this as a variable for this to work?
https://github.com/hmcts/aks-cft-deploy/blob/949fd583a23606e721625cd7492ebff0dac4d8d7/components/aks/inputs-default.tf#L17